decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -30,6 +30,10 @@ class InstallGenerator < Rails::Generators::Base
                                   default: false,
                                   desc: "Don't generate a Gemfile for the application"
 
+      class_option :profiling, type: :boolean,
+                               default: false,
```

Comment:
shouldn't it be `true`?

Reply:
@andreslucena asked for those to only be set for development apps. Otherwise, any new app will have the profiling tools by default and they might not want that.

Reply:
I think it's about whether we prefer to forcefully pass the `--profiling` flag in `generators.rake` so that things are explicit as we do with `--recreate_db` or `--seed-db`. I'd say its better to be consistent with these other flags while at the same time ensuring the test app does not include these gems.

Reply:
@mrcasals thanks for chiming inn, I missed `decidim-dev/lib/tasks/generators.rakeL48` :eyes: